src/cascade/executor/epiviz_runner.py

```diff
@@ -156,8 +161,9 @@ def model_context_from_settings(execution_context, settings):
         MATHLOG.warning(f"removing {remove_cnt} rows from bundle where standard_error == 0.0")
         model_context.input_data.observations = model_context.input_data.observations[mask]
 
-    add_mortality_data(model_context, execution_context)
-    add_omega_constraint(model_context, execution_context)
+    settings.model.drill_sex = 1
```

Comment:
This is stray debugging code?

Reply:
yes! totally a mistake. thank you so much.